### ISSUE TITLE
can't dismiss the captcha modal using "X" or the "ESC" key

### ISSUE THREAD

**Issue by kjozwiak** [brave/browser-laptop#14225]
### Description

Once you've clicked on `Claim my free tokens` and received the captcha modal, you won't be able to dismiss the modal using either `X` or the `ESC` key.

### Steps to Reproduce

  1. launch brave using `LEDGER_VERBOSE=true LEDGER_ENVIRONMENT=staging` (used `0.24.100` de50a5a which was provided by @NejcZdovc)
  2. go into `about:preferences#payments` and enable Brave Payments
  3. click on `Claim my free tokens`
  4. once you get the captcha, attempt to close it via `ESC` or `X`

**Actual result:**

Once the captcha modal is visible, the user can't close it via `X` nor using the `ESC` key.

**Expected result:**

Users should be able to dismiss the captcha modal using either the `X` or using the `ESC` key.

**Reproduces how often:**

100% reproducible using STR mentioned above.

### Brave Version

**about:brave info:**

```
Brave: 0.24.100 
V8: 6.6.346.32 
rev: de50a5a6b058be47b5cf958cd65215793c0a335b 
Muon: 6.0.12 
OS Release: 17.5.0 
Update Channel: Nightly 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 66.0.3359.170
```

**Reproducible on current live release:**

No, the captcha feature is slated to be released in this milestone.

### Additional Information

Reproduced by @mandar-brave as well.

**Comment by NejcZdovc** [comment]
Closed with #14189. This overlays are working only with button and not with ESC

**Comment by btlechowski** [comment]
Verified on Ubuntu 17.10 x64
* 0.22.722 1dde2a2d834ef6cdf21a04be4e9bdacac6e7324f
* muon 6.0.12
* libchromiumcontent 66.0.3359.170
